compiler/rustc_typeck/src/check/method/mod.rs

```diff
@@ -504,6 +525,7 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
                     rhs_span: opt_input_expr.map(|expr| expr.span),
                     is_lit: opt_input_expr
                         .map_or(false, |expr| matches!(expr.kind, hir::ExprKind::Lit(_))),
+                    output_pred: None,
```

Comment:
Not necessary for this PR: I wonder if we can provide the predicate somehow here 🤔